Sources/ElementDiff.swift

```diff
@@ -127,16 +127,16 @@ extension UICollectionView {
 
   /// Update UICollectionView items based on `ElementDiff`
   public func updateSection(section: Int, diff: ElementDiff) {
```

Comment:
I think `updateSection(_ section` is the most consistent with `UITableView.insertSections`